### ISSUE TITLE
I cant Auto-fill Username

### ISSUE THREAD

**Issue by Mioni** [bitwarden/browser#788]
The password goes normal but the user does not auto-fill already tried to use the custom field but without success. 

http://portalateneu.fate.edu.br

**Comment by kspearrin** [comment]
Should be fixed in the next version. You can target that field based on its `aria-label` attribute of "ra".
